projects/L4T/devices/Switch/packages/usb-gadget-scripts/package.mk

```diff
@@ -0,0 +1,35 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#
+#  This Program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2, or (at your option)
+#  any later version.
+#
+#  This Program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.tv; see the file COPYING.  If not, write to
+#  the Free Software Foundation, 51 Franklin Street, Suite 500, Boston, MA 02110, USA.
+#  http://www.gnu.org/copyleft/gpl.html
+################################################################################
+
+PKG_NAME="usb-gadget-scripts"
+#PKG_DEPENDS_TARGET="umtp-responder"
+PKG_SHORTDESC="Nintendo Switch USB Gadget scripts, and configs"
+PKG_TOOLCHAIN="manual"
+
+makeinstall_target() {
+  mkdir -p ${INSTALL}/{usr/bin,usr/lib/systemd/system/}
+    cp ${PKG_DIR}/assets/usb-gadget.sh ${INSTALL}/usr/bin/
+    chmod +x ${INSTALL}/usr/bin/usb-gadget.sh
+    cp ${PKG_DIR}/system.d/usb-gadget.service ${INSTALL}/usr/lib/systemd/system/
+  #Enable Services
+  enable_service usb-gadget.service
+  enable_service usb-tty.service
```

Comment:
move `enable_service` to `post_install()`
maybe this is the reason why it is not working properly?

Reply:
This works fine, post_install() wasnt being run in this package, didnt feel like looking for the reason, if this worked. Possibly it isnt called when package is marked as manual build. ?

Reply:
strange, there is only one condition for calling post_install and that is building package for target. Also copying of files in PKG_DIR/system.d should be automatic.
and again, if the only purpose of a package is to copy files to filesystem, put such files into the filesystem folder of the project / device, the buildsystem takes care of copying the file structure + activation of the services.